src/main/java/io/cos/cas/osf/authentication/support/OsfInstitutionUtils.java

```diff
@@ -53,6 +55,8 @@ public static boolean validateInstitutionForLogin(final JpaOsfDao jpaOsfDao, fin
                 );
             } else if (DelegationProtocol.CAS_PAC4J.equals(delegationProtocol)) {
                 institutionLoginUrlMap.put(institution.getInstitutionId(), institution.getName());
+            } else if (DelegationProtocol.AFFILIATION_VIA_ORCID.equals(delegationProtocol)) {
+                institutionLoginUrlMap.put(DelegationProtocol.AFFILIATION_VIA_ORCID.getId(), institution.getName() + ORCID_SUFFIX);
```

Comment:
Institution whose SSO is done via ORCiD have a suffix " (via ORCiD SSO)" appended to their names.

<img width="600" alt="via-orcid-institution-in-drop-down-list" src="https://user-images.githubusercontent.com/3750414/160059271-993dd3fc-d1f1-4949-a197-5f05b3c48afe.png">